test/serializer/items/attributes_test.rb

```diff
@@ -4,12 +4,12 @@ module CollectionJson
   class Serializer
     class TestAttributes < Minitest::Test
       def setup
-        @user = User.new(name: "Carles Jove", email: "hola@carlus.cat")
+        @user = User.new(name: "Carles Jove", email: "hola@carlus.cat", date_created: "2015-02-01")
         @user_serializer = UserSerializer.new(@user)
       end
 
       def test_attributes_properties
-        assert_equal [:name, :email], @user_serializer.items.attributes
+        assert_equal [:name, :email, {:date_created=>{:prompt=>"Member since"}}], @user_serializer.items.attributes
```

Comment:
Line is too long. [115/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.